assets/javascript/all.js

```diff
@@ -84,7 +84,7 @@ function showHiddenBlock(block) {
     userAgent: navigator.userAgent,
   });
 
-  window.setTimeout(function() {
+  window.setTimeout(function () {
     button.style.display = 'block';
     hiddenData.style.display = 'none';
   }, 3000);
```

Comment:
Looking at https://dsdmoj.atlassian.net/wiki/spaces/HUB/pages/2475295001/Viewing+account+balances+on+The+Content+Hub if we choose/chose option 1 I think we need to remove this timeout

Reply:
This code is the old code which used originally for balances, this is still used on the `Visits` page for displaying the `Visitor Name` so has not been removed. I think the only reason this file has come up in this PR is because of the formatting changes